server/handlers.go

```diff
@@ -0,0 +1,114 @@
+package server
+
+import (
+	"database/sql"
+	"github.com/davecgh/go-spew/spew"
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/status-im/status-go/protocol/identity/identicon"
+	"github.com/status-im/status-go/protocol/images"
+)
+
+type HandlerPatternMap map[string]http.HandlerFunc
+
+func handleIdenticon(logger *zap.Logger) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
```

Comment:
The three media handlers were originally creating structs that took params, these new `HandlerFunc` funcs take params and return handlers. Less overhead, more extensible, easier to implement